### DISCUSSION TITLE
[Enhancement] [form-item] el-form-item组件建议加个tip属性用来传入在校验前显示在下面的提示信息

### DISCUSSION THREAD

**Discussion by TheChosenPerson** [element-plus/element-plus#16256] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `form-item`

## Description

现在：el-form-item在检验后如果有错误信息，将会在下面显示红色的错误信息，并且会将该项变成红色
增强：el-form-item在检验前显示通过tip属性传入的正常颜色的文字，在检验后如果有错误信息，错误信息将会代替tip，并且将该项变成红色，如果没有报错信息，仍然显示通过tip属性传入的正常颜色的提示。
简单来说就是给el-from-item添加一个提示语的功能，如果可以的话，tip的显示方式可以有两种，一种就是像上面描述的那样，和错误信息占用同一块区域，另一种就是在label旁边有个小问号，问号里显示该信息

### Additional comments

非常喜欢这个组件库，一直在使用element-ui和element-plus！

**Comment by L0st1** [comment]
我猜类似于这种？
![image](https://github.com/element-plus/element-plus/assets/64479959/58bd0593-1b4b-4b70-b6fc-0347b0464462)
